CHANGELOG.md

```diff
@@ -1,5 +1,64 @@
 # CHANGELOG
 
+## 2.0.0 (2022-11-14)
+
+MLflow 2.0.0 includes several breaking changes from the 1.x API, additional major features and improvements
+
+Breaking Changes:
+
+- [Recipes] Renamed `mlflow.pipelines` to `mlflow.recipes`
+- [Tracking] Remove deprecated `list` APIs for experiments, models, and runs from Python, Java, JS, and R APIs (#6785, #6786, #6787, #6788, #6800, #6868, @dbczumar)
+- [Tracking] Remove deprecated `runs` data from `get_experiment` REST API response (#6541, #6524 @dbczumar)
+- [Tracking] Remove deprecated `MLflowClient.download_artifacts` API (#6537, @WeichenXu123)
+- [Tracking] Add support for experiment creation resolution and validation for `MLFLOW_EXPERIMENT_NAME` and `MLFLOW_EXPERIMENT_ID` environment variables (#6674, @BenWilson2)
+- [Tracking] Modify tracking server configuration to use `--serve-artifacts` by default (#6502, @harupy)
+- [Tracking] Introduce random `experiment_id` generation for `filestore` tracking server backend to enable threadsafe concurrency (#7070, @BenWilson2)
+- [Tracking] Remove `dataset_name` and `on_data_{name | hash}` suffixes from `mlflow.evaluate` metric keys (#7042, @harupy)
+- [Models] Remove deprecated `mlflow.keras` module and enable `keras` support within `tensorflow` flavor (#6530, @WeichenXu123)
+- [Models] Remove deprecated `mlflow.sklearn.eval_and_log_metrics` API in favor of `mlflow.evaluate` API (#6520, @dbczumar)
```

Comment:
Can we also add a feature entry indicating that the `mlflow.evaluate()` API is now stable?